test/Ported/Server/Replication/PullReplicationTest.ts

```diff
@@ -328,7 +328,7 @@ import { PullReplicationDefinition } from "../../../../src/Documents/Operations/
         } finally {
             sink.dispose();
         }
-    });
+    }).timeout(60_000);
```

Comment:
I remember doing `this.timeout(...)` inside the test func worked in the past. I think that's more explicit like this. What do you think?

Reply:
done